src/components/capacity-breakdown/capacity-breakdown-card.tsx

```diff
@@ -0,0 +1,112 @@
+import { getStackChartStats, sortInstantVectorStats } from '../breakdown-card/utils';
+import { getSelectOptions } from '../breakdown-card/breakdown-dropdown';
+import './capacity-breakdown-card.scss';
+import { humanizeBinaryBytes } from '../../humanize';
+import { BreakdownQueryMapODF } from '../../constants/queries';
+import { PROJECTS, STORAGE_CLASSES, PODS } from '../../constants/index';
+import { 
+  getInstantVectorStats,
+} from '../../selectors/promethues-utils';
+import { parseProps } from '../../selectors/index';
+
+const keys = [PROJECTS, STORAGE_CLASSES, PODS];
+const breakdownSelectItems = getSelectOptions(keys);
+
+const BreakdownCard: React.FC<any> = (props) => {
+  const {name} = parseProps(props);
+  
+  const [metricType, setMetricType] = React.useState(PROJECTS);
+  const [isOpenBreakdownSelect, setBreakdownSelect] = React.useState(false);
+  const { model, metric, queries } = BreakdownQueryMapODF(name, metricType);
+  const queryKeys = Object.keys(queries);
+
+  const [byUsedmetric] = usePrometheusPoll({
+    query: queries[queryKeys[0]],
+    endpoint: "api/v1/query" as any,
+  });
+  
+  const [totalUsedmetric] = usePrometheusPoll({
+    query: queries[queryKeys[1]],
+    endpoint: "api/v1/query" as any,
+  });
+  const metricTotal = _.get(totalUsedmetric, 'data.result[0].value[1]');
+
+  const [usedmetric] = usePrometheusPoll({
+    query: queries[queryKeys[2]],
+    endpoint: "api/v1/query" as any,
+  });
+  const flashsystemUsed =  _.get(usedmetric, 'data.result[0].value[1]');
+
+  const humanize = humanizeBinaryBytes;
+  const top6MetricsData = getInstantVectorStats(byUsedmetric, metric);
+  const top5SortedMetricsData = sortInstantVectorStats(top6MetricsData);
+  const top5MetricsStats = getStackChartStats(top5SortedMetricsData, humanize);
+
+  const handleMetricsChange: SelectProps['onSelect'] = (_e, breakdown) => {
+    setMetricType(breakdown as string);
+    setBreakdownSelect(!isOpenBreakdownSelect);
+  };
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Provisioned Capacity Breakdown</DashboardCardTitle>
+        <div className="flashsystem-capacity-breakdown-card__header">
+          <Select
+            className="flashsystem-capacity-breakdown-card-header__dropdown"
+            autoFocus={false}
+            onSelect={handleMetricsChange}
+            onToggle={() => setBreakdownSelect(!isOpenBreakdownSelect)}
+            isOpen={isOpenBreakdownSelect}
+            selections={[metricType]}
+            placeholderText={metricType}
+            aria-label="Break By Dropdown"
+            isCheckboxSelectionBadgeHidden
+          >
+            {breakdownSelectItems}
+          </Select>
+        </div>
+      </DashboardCardHeader>
+      <DashboardCardBody className="flashsystem-capacity-breakdown-card__body">
+        <BreakdownCardBody
+          isLoading={false}
+          hasLoadError={false}
```

Comment:
Is there a specific reason for assigning `false` to these props ??
We can poll for it as well right ?
```
const [metrics, metricsLoadError, metricsLoading] = usePrometheusPoll({ ... });
```